stop:added confirmation (#929)
tarantool/tt: CHANGELOG.md

### Added

> - ``tt stop`` confirmation prompt.``-y`` and ``tt --no-prompt`` options is added to accept stop

Reviewer: ```suggestion
- `tt stop` confirmation prompt. `-y` option is added to accept stop without prompting.
```
